ProcessMaker/Http/Controllers/AboutController.php

```diff
@@ -120,7 +121,15 @@ private function getNayraMicroServiceAbout(): ?array
             $about = Cache::get('nayra.about', null);
             if (!$about) {
                 // Send about message to receive about information from nayra service
-                MessageBrokerService::sendAboutMessage();
+                try {
+                    MessageBrokerService::sendAboutMessage();
+                } catch (Throwable $e) {
+                    return [
+                        'name' => 'processmaker/nayra-service',
+                        'description' => __('Nayra microservice is not available at this moment.'),
```

Comment:
this text is not in the translations file

Reply:
Fixed